manifests/base/runner-manager/main/service.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: runner-manager
+  labels: {}
+spec:
+  type: ClusterIP
+  ports:
+    - port: 80
```

Let's keep to 8000 for consistency with the deployment.
```suggestion
    - port: 8000
```